actionpack/lib/action_controller/metal/conditional_get.rb

```diff
@@ -13,26 +13,29 @@ module ConditionalGet
     end
 
     module ClassMethods
-      # Allows you to consider additional controller-wide information when generating an ETag.
-      # For example, if you serve pages tailored depending on who's logged in at the moment, you
-      # may want to add the current user id to be part of the ETag to prevent unauthorized displaying
-      # of cached pages.
+      # Allows you to consider additional controller-wide information when
+      # generating an ETag.
+      # For example, if you serve pages tailored depending on who's logged in
+      # at the moment, you may want to add the current user id to be part of
+      # the ETag to prevent unauthorized displaying of cached pages.
       #
       #   class InvoicesController < ApplicationController
       #     etag { current_user.try :id }
       #
       #     def show
-      #       # Etag will differ even for the same invoice when it's viewed by a different current_user
+      #       # Etag will differ even for the same invoice when it's viewed
+      #       # by a different current_user
```

Comment:
what is the change in this paragraph? Please remove the wrapping if there is no wording change.